app/web/components/Select.tsx

```diff
@@ -31,10 +35,12 @@ export default function Select<T extends Record<string | number, string>>({
 }: Omit<SelectProps, "children"> & {
   id: string;
   options: Extract<keyof T, string | number>[];
-  value?: T extends undefined ? string | number : keyof T;
+  // value?: T extends undefined ? string | number : keyof T;
```

Comment:
feedback here pls :)

Reply:
What about `value?: string | number`.

Take a look at the type for the `Select` on line 24 you can see it says `Record<string | number, string>` which gives a clue.

Then I searched `<Select` in VSCode and checked the values of all the Selects being used and they all seem to be a string or number.

Obviously if we ever want to use something besides a string or number we'll need to update it but I think it's good to start strict and expand as we need it.

Reply:
Ooo this is a good question let me think about it. I think we want to make the Select itself more flexible since it's reusable but then the values in the components that use the Select more specific. I'll get back to you tomorrow on this about to board a flight. We should give it a type I just gotta think about which.

Reply:
Okay I got a minute already haha. So if you want to refactor this, I'd do something like this:

https://mariusschulz.com/blog/passing-generics-to-jsx-elements-in-typescript

Keep in mind if you change the value you also need to update the type of line 24 of the `Select`.

I checked the `Select` values and we are always using a `string | number` pretty much, you get a clue of that also on line 24.

I'd say make an `OptionValue` type like in the example above and use that. I think you can do it almost exactly as in that guide ^^.

It's going to cause errors for other types that are any or vague in other components like `FilterDialog`, but a lot of those should be fixed once the linting PR is merged.

Reply:
If you're not gonna update this in this PR btw, I'd put it back.

Reply:
wait, I still didn't read the link you sent me, I just need more time 😅